frontend/src/components/keyboard/index.js

```diff
@@ -17,13 +17,19 @@ const Keyboard = observer(() => {
     const keyboardRef = useRef(null);
 
     const onKeyPress = (button) => {
-        if (button === '{capslock}') {
+        if (button === '{capslock}' || button === '{shiftleft}' || button === '{shiftright}') {
             setLayoutName(layoutName === 'default' ? 'shift' : 'default')
         }
         const key = syntethicKeysMap[button] || button
         handleKeyboardShortcuts(key)
     }
 
+    const onKeyReleased = (button) => {
+      if (button === '{capslock}' || button === '{shiftleft}'|| button === '{shiftright}') {
```

Comment:
It's worth moving this logic to a separate method like `changeLayout(name, button)` to avoid duplication,

Reply:
done